eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/HTTPClientHandler.java

```diff
@@ -95,7 +95,8 @@ void delete(HttpExchange httpExchange) throws IOException {
             Error error = new Error(e.toString(), stackTrace);
             String result = JsonUtils.toJSONString(error);
             httpExchange.sendResponseHeaders(500, result.getBytes().length);
```

Comment:
Please help simplify, replace with 'httpExchange.sendResponseHeaders(500,  0);'

Reply:
Sure, updated in latest commit.